contracts/internals/tokenWhitelist.sol

```diff
@@ -181,12 +193,33 @@ contract TokenWhitelist is ENSResolvable, Controllable, Ownable, Claimable {
         return (stablecoinInfo.symbol, stablecoinInfo.magnitude, stablecoinInfo.rate, stablecoinInfo.available, stablecoinInfo.loadable, stablecoinInfo.burnable, stablecoinInfo.lastUpdate);
     }
 
-    /// @notice This returns an array of our whitelisted address
-    /// @return address[] of our whitelisted tokens
+    /// @notice This returns an array of all whitelisted token addresses
+    /// @return address[] of whitelisted tokens
     function tokenAddressArray() external view returns (address[] memory) {
         return _tokenAddressArray;
     }
 
+    /// @notice This returns an array of all burnable token addresses
+    /// @return address[] of burnable tokens
+    function burnableTokens() external view returns (address[] memory) {
```

Comment:
we could consider storing an array of burnable tokens instead, but this works ...

Reply:
yes, this is true and we need to do it for scaling reasons, that's why we are just exposing an interface now to the holder contract, so we can replace the tokenWhitelist anytime without needing to change the Holder contract